.gitignore

```diff
@@ -21,3 +21,6 @@
 /yarn-error.log
 
 .byebug_history
+
+# Ignore application configuration
+/config/application.yml
```

Comment:
I know figaro discourages committing anything but we typically like to have a .sample file that documents what needs to go in the not committed file. In this case there would be a `config/application.yml.sample`. I'm not sure what figaro would say about this.

Reply:
I can do that, but I don't understand what would be included in that file. I used Figaro as I couldn't decipher how to do the process manually.

Reply:
The sample file would have every element of the actual file but with fake values.

Reply:
ok, done